test/test.js

```diff
@@ -7,21 +7,25 @@ var path = require("path")
 var assert = require("assert")
 
 // Require all visualizations
-glob.sync("./dist/*.js" ).forEach(function( file ) {
-  require( path.resolve( file ) );
-});
+glob.sync("./dist/*.js").forEach((file) => {
+  require(path.resolve(file))
+})
 
-looker.plugins.visualizations.all().forEach((vis) =>{
+looker.plugins.visualizations.all().forEach((vis) => {
   describe(`${vis.label} (as ${vis.id})`, () => {
-    it("should load and not use unavailable things", () => { assert(true) });
+
+    it("should load and not use unavailable things", () => {
+      // TODO what do we need to assert here
+      assert(true)
+    });
```

Comment:
Are tests forthcoming still?

Reply:
@wynnandrewj We should probably talk about this.  We are not currently working on additional tests...